rest-client/mhs/spine/async_express.http

```diff
@@ -1,6 +1,6 @@
 
 # @name Async_Express
-POST https://192.168.128.11/reliablemessaging/queryrequest HTTP/1.1
+POST {{SPINE_HOSTNAME}}/reliablemessaging/queryrequest HTTP/1.1
```

Comment:
hot hostname but "base_url"
Hostname would be 192.168.128.11 only (kind of)
Here you expect domain and protocol